app/models/person.rb

```diff
@@ -58,8 +59,31 @@ def mentor
     ProtocolSubscription.where(filling_out_for_id: id).where.not(person_id: id).first&.person
   end
 
+  def stats(week_number, year, threshold_percentage)
+    person_completed = 0
+    person_total = 0
+    protocol_subscriptions.each do |subscription|
+      past_week = subscription.responses.in_week(week_number: week_number, year: year)
+      person_completed += past_week.completed.count || 0
+      person_total += past_week.count || 0
+    end
+    {
+      met_threshold_completion: check_threshold(person_completed, person_total, threshold_percentage),
+      completed: person_completed,
+      total: person_total
+    }
+  end
+
   private
 
+  def check_threshold(completed, total, threshold_percentage)
+    return 0 unless total.positive?
+    threshold_percentage ||= Person::DEFAULT_PERCENTAGE
+    threshold_percentage = threshold_percentage.to_i
+    actual_percentage = completed.to_d / total.to_d * 100
+    actual_percentage >= threshold_percentage ? 1 : 0
```

Comment:
Hier zeg je `>=` maar volgens mij had je in je view `>` staan. Moet dat niet hetzelfde zijn?